### PR TITLE
Updated Model Builder docs based on new release

### PR DESCRIPTION
**Pull request by luisquintanilla**
## Summary

- [x] Updated price prediction tutorial
- [x] Updated sentiment analysis tutorial

[Price Prediction Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/predict-prices-with-model-builder?branch=pr-en-us-14679)
[Sentiment Analysis Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis-model-builder?branch=pr-en-us-14679)

[AB#1572728](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1572728)

### CODE REVIEW

File: docs/machine-learning/tutorials/predict-prices-with-model-builder.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Each row in the `taxi-fare-train.csv` data set contains details of trips made by
     - **vendor_id:** The ID of the taxi vendor is a feature.
     - **rate_code:** The rate type of the taxi trip is a feature.
     - **passenger_count:** The number of passengers on the trip is a feature.
-    - **trip_time_in_secs:** The amount of time the trip took.
+    - **trip_time_in_secs:** The amount of time the trip took. ou want to predict the fare of the trip before the trip is completed. At that moment you don't know how long the trip would take. Thus, the trip time is not a feature and you'll exclude this column from the model.
```
Comment (natke) [inline docs/machine-learning/tutorials/predict-prices-with-model-builder.md]: Typo? ou --> You

File: docs/machine-learning/tutorials/sentiment-analysis-model-builder.md [hunk 2]
```diff
@@ -113,23 +114,13 @@ Two projects will be created as a result of the training process.
     The following projects should appear in the **Solution Explorer**:
 
     - *SentimentRazorML.ConsoleApp*: A .NET Core Console application that contains the model training and prediction code.
-    - *SentimentRazorML.Model*: A .NET Standard class library containing the data models that define the schema of input and output model data as well as the persisted version of the best performing model during training.
+    - *SentimentRazorML.Model*: A .NET Standard class library containing the data models that define the schema of input and output model data as well as the serialized version of the best performing model during training.
```
Comment (natke) [inline docs/machine-learning/tutorials/sentiment-analysis-model-builder.md]: Suggestion: serialized -> saved?

Comment (luisquintanilla) [reply docs/machine-learning/tutorials/sentiment-analysis-model-builder.md]: #resolved

File: docs/machine-learning/tutorials/predict-prices-with-model-builder.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/predict-prices-with-model-builder.md]: #resolved

**Review by natke — APPROVED** [review]
LGTM!

**Pull request merged**